vspace/src/main/webapp/WEB-INF/views/exhibition/space.html

```diff
@@ -457,11 +494,11 @@
     		        	 moduleLinkURL = [[@{|/exhibit/${space?.id}/module/|}]]+link.link.module.id;
     		        }
     		        moduleLink.attr("href", moduleLinkURL);
-    		        var linkDisplay = $('<div class="InfoModule_'+index+' Info_cz_Class"><svg class="Ellipse_8_c"><ellipse fill="rgba(222,222,222,1)" class="Ellipse_8_c_Class" rx="14.5" ry="14.5" cx="14.5" cy="14.5"></ellipse></svg><svg class="Ellipse_10_c"><ellipse fill="rgba(240,240,240,1)" class="Ellipse_10_c_Class" rx="12.5" ry="12.5" cx="12.5" cy="12.5"></ellipse></svg><svg class="Ellipse_9_c"><ellipse fill="rgba(255,255,255,1)" class="Ellipse_9_c_Class" rx="10.5" ry="10.5" cx="10.5" cy="10.5"></ellipse></svg><span class="icon-info Icon_awesome_info_m"></span><span class="tooltiptext">'+link.link.name+'</span></div>');
+    		        var linkDisplay = $('<div class="InfoModule_'+index+' Info_cz_Class"><svg class="Ellipse_8_c"><ellipse fill="rgba(222,222,222,1)" class="Ellipse_8_c_Class" rx="14.5" ry="14.5" cx="14.5" cy="14.5"></ellipse></svg><svg class="Ellipse_10_c"><ellipse fill="rgba(240,240,240,1)" class="Ellipse_10_c_Class" rx="12.5" ry="12.5" cx="12.5" cy="12.5"></ellipse></svg><svg class="Ellipse_9_c"><ellipse fill="rgba(255,255,255,1)" class="Ellipse_9_c_Class" rx="10.5" ry="10.5" cx="10.5" cy="10.5"></ellipse></svg><span class="icon-move Icon_awesome_info_m"></span><span class="tooltiptext">'+link.link.name+'</span></div>');
     		        if(link.type=="IMAGE" && link.image != null){
                         var moduleImageURL = [[@{/api/image/}]]+link.image.id;
                         linkDisplay = $('<img class="InfoModule_'+index+'" src="'+moduleImageURL+'" />');
-    		        } else if(defaultModuleImageFlag === true){
+    		        } else if(defaultModuleImageFlag === true && defaultModuleImageDisableFlag !== true){
```

Comment:
see, here it gets really confusing. When is `defaultModuleImageDisableFlag` true? when the image is disabled or not? should be `defaultModuleImageIsDisabled` or `defaultModuleImageDisabled`, then it's clear when that is true or false.